pkg/utils/utils.go

```diff
@@ -115,21 +113,20 @@ func createMaps() {
 	ResourceGetterMap["clusterrolebindings"] = KubeClient.RbacV1().RESTClient()
 
 	// Allowed event kinds map
-	for _, r := range config.Resources {
-		for _, ns := range r.Namespaces {
-			if r.Name == "ingresses" {
-				AllowedEventKindsMap[EventKind{strings.TrimSuffix(r.Name, "es"), ns}] = true
+	for _, r := range botkubeConf.Resources {
+		for _, e := range r.Events {
+			if e != config.ErrorEvent {
```

Comment:
Are we directly ignoring events which are not errortype ?

Reply:
Yes, other are `info` events of which we already covered important ones in lifecycle events.